__tests__/cmds/versions.test.js

```diff
@@ -144,29 +140,25 @@ describe('rdme versions*', () => {
   });
 
   describe('rdme versions:delete', () => {
-    it('should error if no api key provided', () => {
-      expect.assertions(1);
-      return deleteVersion.run({}).catch(err => {
-        expect(err.message).toBe('No project API key provided. Please use `--key`.');
-      });
+    it('should error if no api key provided', async () => {
+      await expect(deleteVersion.run({})).rejects.toThrow('No project API key provided. Please use `--key`.');
     });
 
-    it('should delete a specific version', () => {
-      const mockRequest = nock(config.host)
+    it('should delete a specific version', async () => {
+      const mockRequest = nock(config.get('host'))
         .delete(`/api/v1/version/${version}`)
         .basicAuth({ user: key })
-        .reply(200)
+        .reply(200, { removed: true })
```

Comment:
Added `{removed: true}` because that's what we return in our API and the `handleRes` handler was failing without this because `.reply(200)` by itself doesn't return JSON for `res.json()`.

We should probably actually return a 204 No Content instead for these delete calls but 🤷